elephant/statistics.py

```diff
@@ -239,18 +247,31 @@ def lv(v):
     """
     # convert to array, cast to float
     v = np.asarray(v)
+    
+    # ensure the input ia a vector
+    if len(v.shape) > 1: # adeed by me for symmetry with CV
+        raise AttributeError("Input shape is larger than 1. Please provide "
+                             "a vector as an input.")
 
     # ensure we have enough entries
     if v.size < 2:
-        raise AttributeError("Input size is too small. Please provide "
-                             "an input with more than 1 entry.")
+        if with_nan:
+           warnings.warn("Input size is too small. Please provide "
+                         "an input with more than 1 entry. lv returns 'NaN'"
+                         "since the argument `with_nan` is True")
```

Comment:
Can you put around all the `True` and `False` keywords  " \` \`". This ensures a better readability.